Utils/Dataflow/pyDKB/dataflow/stage/AbstractStage.py

```diff
@@ -29,7 +29,22 @@ def __init__(self, description="DKB Dataflow stage"):
         * ...
         """
         self.ARGS = None
-        self.__parser = argparse.ArgumentParser(description=description)
+        self.__parser = argparse.ArgumentParser(
+            formatter_class=argparse.RawDescriptionHelpFormatter,
+            description=description,
+            epilog='''\
+                Shortcut table:
+                
+                mode | source | dest | eom | eop
+                -----+--------+------+-----+-----
+                  s  |    s   |   s  |  \\n |  \\0
+                -----+--------+------+-----+-----
+                  f  |   f/h  |  f/h |  \\n |
+                -----+--------+------+-----+-----
+                  m  |   s/h  |   s  |  \\n |
+                -----+--------+------+-----+-----
+                  h  |    h   |  h/f |  \\n |
+                -----+--------+------+-----+-----''')
```

Comment:
As you said, it looks a bit misplaced in the full help message:
```
>>> parser.print_help()
usage: [-h] [-m [MODE]] [-e [EOM]] [-E [EOP]]

Stage description

optional arguments:
  -h, --help            show this help message and exit
  -m [MODE], --mode [MODE]
                        Processing mode: (f)ile, (s)tream or (m)ap-reduce
                        (default: f).
  -e [EOM], --end-of-message [EOM]
                        Custom end of message marker.
  -E [EOP], --end-of-process [EOP]
                        Custom end of process marker.

                Shortcut table:
                
                mode | source | dest | eom | eop
                -----+--------+------+-----+-----
                  s  |    s   |   s  |  \n |  \0
                -----+--------+------+-----+-----
                  f  |   f/h  |  f/h |  \n |
                -----+--------+------+-----+-----
                  m  |   s/h  |   s  |  \n |
                -----+--------+------+-----+-----
                  h  |    h   |  h/f |  \n |
                -----+--------+------+-----+-----
```

I think it would really be better to move it to the left edge and add a couple of words to explain what are we talking about, to get something like this:
```
>>> parser.print_help()
usage: [-h] [-m [MODE]] [-e [EOM]] [-E [EOP]]

Stage description

optional arguments:
  -h, --help            show this help message and exit
  -m [MODE], --mode [MODE]
                        Processing mode: (f)ile, (s)tream or (m)ap-reduce
                        (default: f).
  -e [EOM], --end-of-message [EOM]
                        Custom end of message marker.
  -E [EOP], --end-of-process [EOP]
                        Custom end of process marker.

NOTES

Processing mode
    is defined as a combination of data source and destination type (local/HDFS
    file(s) or standard stream) and pre-defined EOP and EOM markers:

    mode | source | dest | eom | eop
    -----+--------+------+-----+-----
      s  |    s   |   s  |  \n |  \0
    -----+--------+------+-----+-----
      f  |   f/h  |  f/h |  \n |
    -----+--------+------+-----+-----
      m  |   s/h  |   s  |  \n |
    -----+--------+------+-----+-----
      h  |    h   |  h/f |  \n |
    -----+--------+------+-----+-----
```